features/provider/admin/applications/create_application.feature

```diff
@@ -0,0 +1,93 @@
+@javascript
+Feature: Create application from Audience
+  In order to control the way my buyers are using my API
+  As a provider
+  I want to create their applications
+
+  Background:
+    Given a provider "foo.3scale.localhost"
+    And a buyer "bob" signed up to provider "foo.3scale.localhost"
+    And buyer "bob" has no applications
+    And a default application plan "Basic" of provider "foo.3scale.localhost"
+    And current domain is the admin domain of provider "foo.3scale.localhost"
+    Then I log in as provider "foo.3scale.localhost"
+
+  Scenario: Create a an application
+    When I create an application "My App" from the audience context
+    Then I should be on the provider side "My App" application page
+    And should see "Application was successfully created"
+    And buyer "bob" should have 1 cinstance
+
+  Scenario: Create an application in single application mode
+    Given provider "foo.3scale.localhost" has multiple applications disabled
```

Comment:
This has nothing to do with the new cucumber. If you still use that syntax please let those cucumber remain in the `old` directory

Reply:
OK in the end I'll move them to old, I think it's more important to merge this.